### PR TITLE
[E2E] Fix the seeding process using DTS

### PR DESCRIPTION
**Pull request by Convly**
Made in collaboration with @alexandrebodin :rocket: 

Fix the seed/import process using data transfer for the e2e tests.

Modified the admin transfer token service's methods to allow defining a token with a custom accessKey (in order to provide consistency for testing env).

Added a constant containing a (non-secret) transfer token shared by the transfer process and the app template.

### CODE REVIEW

File: e2e/app-template/template/src/constants.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const CUSTOM_TRANSFER_TOKEN_ACCESS_KEY =
```
Comment (Convly) [inline e2e/app-template/template/src/constants.js:3]: This looks like a secret but really isn't. It's only a randomly generated string used for testing purpose :slightly_smiling_face:

**Comment by joshuaellis** [comment]
I like this, a lot

File: e2e/app-template/template/src/constants.js (hunk 1 above)
Comment (gu-stav) [reply e2e/app-template/template/src/constants.js:3]: Can we make this a code comment? :)

**Review by gu-stav — APPROVED** [review]
I haven't tested it yet, but code-wise it looks good to me. Did you give it a spin already?

File: e2e/app-template/template/src/constants.js (hunk 1 above)
Comment (innerdvations) [reply e2e/app-template/template/src/constants.js:3]: Making it all zeros would work too and make it obvious it's a test value

Comment (Convly) [reply e2e/app-template/template/src/constants.js:3]: We can do that indeed

Comment (Convly) [reply e2e/app-template/template/src/constants.js:3]: I'll update that tomorrow

**Comment by derrickmehaffy** [comment]
Really nice!

File: e2e/app-template/template/src/constants.js (hunk 1 above)
Comment (Convly) [reply e2e/app-template/template/src/constants.js:3]: Done :slightly_smiling_face:

**Comment by Convly** [comment]
> I haven't tested it yet, but code-wise it looks good to me. Did you give it a spin already?

Yes, here's what it looks like for now

[e2e-dts-test.webm](https://user-images.githubusercontent.com/25851739/233576478-79f73e46-749c-4331-83e5-d4938bceed18.webm)

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**